frontend/htmlTemplate.ts

```diff
@@ -101,6 +101,10 @@ export default ({
                         browserId: getCookieValue('bwid')
                     };
                 })(window, document);
+                (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+                    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+                    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```

Comment:
We should test whether this or the async script is better.

Reply:
Yeah. I think time to interactive is the metric that will be most useful

Reply:
I've added a [card](https://trello.com/c/O6SZe3NM) to investigate third party script loading strategies